app/javascript/leftSidebar/TagsFollowed.jsx

```diff
@@ -12,6 +12,7 @@ export const TagsFollowed = ({ tags = [] }) => {
   return (
     <Fragment>
       {tags.map(({ name, id, points }) =>
+        // Note points are the calculated points of a Tag.  See Tag#points for more discussion.
```

Comment:
Is this intended to say "NOTE: points" or did I overlook that we have a concept called "note points"?

Reply:
Intended to say "NOTE: points" correcting my comments.